test/e2e/inspector-e2e.test.js

```diff
@@ -0,0 +1,106 @@
+import path from 'path';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { startServer as startAppiumFakeDriverServer } from 'appium-fake-driver';
+import _ from 'lodash';
+import { retryInterval } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const FAKE_DRIVER_PORT = 12121;
+
+const TEST_APP = path.resolve('node_modules', 'appium-fake-driver', 'test', 'fixtures', 'app.xml');
+
+const DEFAULT_CAPS = {
+  platformName: 'Fake',
+  deviceName: 'Fake',
+  app: TEST_APP,
+};
+
+const dcaps = _.toPairs(DEFAULT_CAPS);
+
+let client;
+let defaultUrl;
+
+async function closeNotification (client) {
+  try {
+    await retryInterval(5, 500, () => {
+      client.click('.ant-notification-notice-close');
+    });
+  } catch (ign) { }
+}
+
+describe('inspector window', function () {
+  before(async function () {
+    // Start an Appium fake driver server
+    startAppiumFakeDriverServer(FAKE_DRIVER_PORT, '127.0.0.1');
+
+    // Navigate to session URL
+    client = this.app.client;
+    const url = await client.url();
+    defaultUrl = url.value;
+    await client.url(defaultUrl + 'session');
+
+    // Set the desired capabilities
+    await client.waitForExist('#btnAddDesiredCapability');
+    for (let i=0; i<dcaps.length - 1; i++) {
+      await client.click('#btnAddDesiredCapability');
```

Comment:
could just move the value entering in here:

```js
await client.setValue(`#desiredCapabilityName_${i}`, dcaps[i][0]);
await client.setValue(`#desiredCapabilityVAlue_${i}`, dcaps[i][1]);
```

to avoid having 2 loops